client/src/components/App/Content/LandingPage/adminLandingPage/investigationsInfo/investigationsInfo.tsx

```diff
@@ -1,50 +1,91 @@
-import React from 'react';
-import { useHistory } from 'react-router-dom';
+import React, { useMemo } from 'react';
 import NavigateBeforeIcon from '@material-ui/icons/NavigateBefore';
-import { Card, CardContent, Grid, IconButton, Typography } from '@material-ui/core';
+import { CardContent, Grid, IconButton, Typography } from '@material-ui/core';
 
-import { landingPageRoute } from 'Utils/Routes/Routes';
 import InvestigationChart from 'models/InvestigationChart';
+import FilterRulesVariables from 'models/FilterRulesVariables';
+import statusToFilterConvertor from 'commons/statusToFilterConvertor';
+import FilterRulesDescription from 'models/enums/FilterRulesDescription';
+import { InvesitgationInfoStatistics } from 'models/InvestigationStatistics';
 
-import useStyles from './investigationsInfoStyles';
+import LoadingCard from '../LoadingCard/LoadingCard';
+import useStyles, { cardWidth, cardHeight } from './investigationsInfoStyles';
 import InvestigationBarChart from './InvestigationBarChart/InvestigationBarChart';
-import InvestigationChartData from './InvestigationBarChart/InvestigationChartData';
 import InvestigationInfoButton from './investigationInfoButton/investigationInfoButton';
 
-const InvestigationsInfo: React.FC = (): JSX.Element => {
+const convertorsToGraph: { [T in keyof InvesitgationInfoStatistics]: Omit<InvestigationChart, 'value'>} = {
+    newInvestigations: {
+        id: FilterRulesDescription.NEW,
+        color: '#1F78B4'
+    },
+    inProcessInvestigations: {
+        id: FilterRulesDescription.IN_PROCESS,
+        color: 'grey'
+    },
+    unassignedInvestigations: {
+        id: FilterRulesDescription.UNASSIGNED,
+        color: '#33A02C'
+    },
+    inactiveInvestigations: {
+        id: FilterRulesDescription.INACTIVE,
+        color: '#F95959'
+    }
+}
+
+const InvestigationsInfo: React.FC<Props> = (props: Props): JSX.Element => {
     const classes = useStyles();
 
-    let history = useHistory();
+    const { onInfoButtonClick, investigationsStatistics, allInvestigationsCount, isLoading } = props;
+
+    const investigationsGraphData = useMemo<InvestigationChart[]>(() => {
+        const returnedArray:InvestigationChart[] = [];
+        Object.keys(convertorsToGraph).forEach((convertor) => {
```

Comment:
You can use map instead of forEach with push.

Reply:
fixed